resources/assets/tconstruct/models/block/stenciltable.json

```diff
@@ -1,11 +1,184 @@
 {
-    "parent": "tconstruct:block/table",
-    "textures": {
-        "particle": "tconstruct:blocks/stenciltable_top",
-        "top": "tconstruct:blocks/stenciltable_top",
-        "side": "tconstruct:blocks/table_side",
-        "bottom": "minecraft:blocks/planks_oak",
-        "leg": "minecraft:blocks/planks_oak",
-        "legBottom": "minecraft:blocks/planks_oak"
-    }
+	"parent": "block/block",
+	"textures": {
+		"0": "tconstruct:chest/stencil_top",
```

Comment:
Don't use numbers for textures, use names so its easier to understand the files. Half the time I edit these by hand